src/hook_files/cli.sh

```diff
@@ -14,14 +14,14 @@ allowPrereleaseCliVersion={{MINIMUM_ALLOW_PRE}}
 noConfigFileExitCode={{NO_CONFIG_FILE_EXIT_CODE}}
 
 upgradeRustyHookCli() {
-  echo "Upgrading rusty-hook cli..."
-  echo "This may take a few seconds..."
+  echo "[rusty-hook] Upgrading rusty-hook cli..."
```

Comment:
Ahhh well done. I'd forgotten about the hooks but yes we definitely want to update these too, thanks!